internal/keycloakb/logging.go

```diff
@@ -0,0 +1,9 @@
+package keycloakb
+
+// Logger interface for logging with level
+type Logger interface {
```

Comment:
Should be defined in common-service ???

Reply:
In Go, the interface should be defined where you are using it, not in the library so I think it is ok to declare it there.